### PR TITLE
Feature/translations.fr/may.2022

### PR DESCRIPTION
**Pull request by slruder**
These are my changes for French translation from today. I still have many to do. This is my first try at a pull request.

### CODE REVIEW

**Review by smattingly — COMMENTED** [review]
@slruder I marked one item that looks doubtful to me. Also, I wanted to confirm that we want to merge all those empty strings that replaced MISSING items.

File: packages/webapp/public/locales/fr/common.json [hunk 1]
```diff
@@ -19,13 +19,13 @@
...
   "GOT_IT": "J'ai compris !",
-  "HERE": "MISSING",
+  "HERE": "Ici",
   "HIDE": "Cacher",
   "HOURS": "Heurs",
   "LOADING": "Chargement en cours...",
   "MARK_ABANDON": "Abandonner",
   "MARK_COMPLETE": "Marquer Complète",
-  "MARK_COMPLETED": "Marquer Complèté",
+  "MARK_COMPLETED": "Marquerpackages/webapp/public/locales/fr/common.json Complèté",
```
[the remark this answers is not in the document]
Comment (slruder) [reply packages/webapp/public/locales/fr/common.json:28]: Thank you for catching that. I fixed it and will push through the change

[the remark this answers is not in the document]
Comment (slruder) [reply packages/webapp/public/locales/fr/common.json:28]: I am not sure what "merge all those empty strings that replaced MISSING items" means. Meeting with David to clarify.

File: packages/api/src/templates/locales/fr.json [hunk 2]
```diff
@@ -1,101 +1,101 @@
...
-    "LOG_IN": "Log In",
+    "JOIN": "Rejoindre",
+    "LOG_IN": "Connexion",
     "OWNER": "Propriétaire",
     "EXTENSION_OFFICER": "Agent de vulgarisation",
     "WORKER": "Ouvrier",
     "MANAGER": "Gestionnaire"
   },
   "EXPORT": {
     "HI": "EXPORT.HI",
```
Comment (smattingly) [inline packages/api/src/templates/locales/fr.json:93]: I think the values in this block should have ordinary French text in them.

File: packages/webapp/public/locales/en/translation.json [hunk 3]
```diff
@@ -65,7 +65,7 @@
     "HOW_MUCH_IS_HARVESTED": "How much is being harvested?",
     "HR": "/hr",
     "MANAGE_CUSTOM_TASKS": "Manage custom tasks",
-    "NEED_MANAGEMENT_PLAN": "You’ll need an active or planned crop plan before you can schedule a harvest task or transplant task. Go to the crop catalogue to create a plan now.",
+    "NEED_MANAGEMENT_PLAN": "You'll need an active or planned crop plan before you can schedule a harvest task or transplant task. Go to the crop catalogue to create a plan now.",
```
Comment (smattingly) [inline packages/webapp/public/locales/en/translation.json:68]: I assume it's no problem, but I'll point out that the more exotic apostrophe characters that were in the files have been replaced with the mundane standard keyboard single quote. May have been done automatically by editor software.

File: packages/webapp/public/locales/fr/translation.json [hunk 4]
```diff
@@ -1,14 +1,14 @@
 {
   "ADD_FARM": {
     "ADDRESS_IS_REQUIRED": "Address civique est requise",
-    "DISABLE_GEO_LOCATION": "",
+    "DISABLE_GEO_LOCATION": "Les services de localisation doivent être activés pour trouver votre position actuelle",
```
Comment (smattingly) [inline packages/webapp/public/locales/fr/translation.json:4]: There are three empty string values in this file.
(Nothing to do with this line; I can only put review comments on lines that were changed in the last update.)

**Review by smattingly — CHANGES_REQUESTED** [review]
@slruder @dtrapplitefarm I found a few things that I pointed out with review comments.

**Review by smattingly — APPROVED** [review]

**Pull request merged**